### ISSUE TITLE
Create a 0.17.0.post1 release with the pyproject.toml file so that installing gammapy works in fresh environments

### ISSUE THREAD

**Issue by maxnoe** [gammapy/gammapy#3026]
In #2896 the pyproject.toml file was added so that gammapy can be installed in fresh environments without having to install Cython manually.

Could you backport this to the 0.17 version, so that that version of gammapy can also be installed like the current master now can?

**Comment by adonath** [comment]
Thanks @MaxNoe, I agree this is unfortunate. However at some point we decided we will not do bugfix releases before v1.0 and instead just release another minor version. The v0.18 release is scheduled in ~3 weeks from now. I feel like an additional call to `pip install cython` does not justify to break the release strategy we had so far...especially because I think the pip install was broken since v0.14, when we removed `astropy_helpers`.  I hope that's fine with you...
